pkgs/top-level/python-packages.nix

```diff
@@ -1543,12 +1541,11 @@ in {
     '';
 
     meta = {
-      homepage = https://aws.amazon.com/cli/;
-      description = "Unified tool to manage your AWS services";
-      license = stdenv.lib.licenses.asl20;
-      maintainers = with maintainers; [ muflax ];
-    };
-  };
+        homepage = https://aws.amazon.com/cli/;
+        description = "Unified tool to manage your AWS services";
+        license = stdenv.lib.licenses.asl20;
+        maintainers = with maintainers; [ muflax ];
+    };                                                                                                                                     };          
```

Comment:
the second parenthesis at the end of this line should got to the to the next line. You have seem to merge two lines by accident.